src/domain/app/footer/__tests__/Footer.test.tsx

```diff
@@ -20,7 +20,7 @@ beforeEach(() => {
 const renderComponent = (route = '/fi') =>
   render(<Footer />, { routes: [route] });
 
-test('matches snapshot', async () => {
+test.skip('matches snapshot', async () => {
```

Comment:
The test was skipped because it was testing Swedish footer and Swedish language is not in use at the moment. 
Added a comment to explain why it is skipped.

Reply:
👍